src/HasMedia/HasMediaTrait.php

```diff
@@ -63,7 +63,7 @@ public function addMedia($file)
      *
      * @return \Spatie\MediaLibrary\FileAdder\FileAdder
      */
-    public function addMediaFromRequest(string $key)
+    public function addMediaFromRequest($key)
```

Comment:
Anything *without* a default value can be left alone. They aren't causing a problem.